packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -167,6 +168,131 @@ def vip_workload_test(dcos_api_session, container, vip_net, proxy_net, named_vip
     return (vip, hosts, cmd, origin_app, proxy_app)
 
 
+def pod_vip_app(network: marathon.Network, host: str, vip: str):
+    ''' Creates a single app in docker container in pod
```

Comment:
Maybe:
"Create an app with a single container in a pod"?

pods are always UCR containers.